class.jetpack.php

```diff
@@ -7249,6 +7249,15 @@ public static function show_backups_ui() {
 		return self::is_plugin_active( 'vaultpress/vaultpress.php' ) || apply_filters( 'jetpack_show_backups', true );
 	}
 
+	/**
+	 * Returns a boolean for whether the Setup Wizard should be displayed or not.
+	 *
+	 * @return bool True if the Setup Wizard should be displayed, false otherwise.
+	 */
+	public static function show_setup_wizard() {
+		return apply_filters( 'jetpack_show_setup_wizard', false ) && self::is_active();
+	}
+
```

Comment:
What do you think about keeping everything in `_inc/lib/admin-pages/class.jetpack-react-page.php`? We most likely won't need it elsewhere for now? If we can avoid adding to the monster that is `class.jetpack.php` for now, I think that will be nice :)

Reply:
Sounds good to me, done!